### PR TITLE
Deliver_now-DeliverLater-AdminController

### LINKED ISSUE #9310: Change deliver_now to deliver_later
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 Part of #6126 Change deliver_now to deliver_later
- [ ] https://github.com/publiclab/plots2/blob/876d0fc084064aaecc23f8003630d7d1ab858fa1/app/controllers/admin_controller.rb

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by BondaSaim**
Fixes #9310
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by BondaSaim** [comment]
Hello @publiclab/reviewers,
My PR is failing unit test with the following error:

FAIL TagSelectionTest#test_graph (0.35s)
        Expected: 18
          Actual: 23
        test/unit/tag_selection_test.rb:14:in `block in <class:TagSelectionTest>'

Could you help me in solving it as I am new to ruby

**Comment by RuthNjeri** [comment]
Thanks for working on this @BondaSaim, the failed test is common and we documented it here https://github.com/publiclab/plots2/issues/9324

I have rerun your tests, hopefully it should pass.

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -60,22 +60,22 @@ def reset_user_password
...
+           .where(status: [1, 4])
        end
-     end 
-   else 
-     # unauthorized. instead of return ugly 403, just send somewhere else 
-     redirect_to '/dashboard' 
+     end
+   else
+     # unauthorized. instead of return ugly 403, just send somewhere else
+     redirect_to '/dashboard'
```
Comment (RuthNjeri) [inline app/controllers/admin_controller.rb:78]: The tests are passing now @BondaSaim 🎉 

Are these changes needed? I am assuming that line 208 is the required change?

Comment (BondaSaim) [reply app/controllers/admin_controller.rb:78]: Thank you @RuthNjeri.
Yes, The change at line 208 is the required one, the other are auto generated by my IDE

**Comment by BondaSaim** [comment]
Hi @RuthNjeri,
If possible, could you kindly look into #9340 and #9341 as I am facing issues over there ?

Thanks a lot in advance

**Comment by cesswairimu** [comment]
Hi @BondaSaim , we normally have new contributors take up only one issue labelled `first-timer-only` and move to `help-wanted` issues so that we can leave the rest  to new contributors...kindly browse through on our `help-wanted` issues if you would like to work on other issues
We will merge this as your first-timer-issue pull request and I will close your other open PRs for first-timer-only issues and free them for others. Thanks for understanding

**Comment by BondaSaim** [comment]
> Hi @BondaSaim , we normally have new contributors take up only one issue labelled `first-timer-only` and move to `help-wanted` issues so that we can leave the rest to new contributors...kindly browse through on our `help-wanted` issues if you would like to work on other issues
> We will merge this as your first-timer-issue pull request and I will close your other open PRs for first-timer-only issues and free them for others. Thanks for understanding

Thank you @cesswairimu for your reply,
I worked on them as no one has taken it up,
I totally understand its purpose and I will work on ```help-wanted``` issues from now
Thank you

File: app/controllers/admin_controller.rb (hunk 1 above)
Comment (RuthNjeri) [reply app/controllers/admin_controller.rb:78]: @BondaSaim can you revert these changes to what was there before? Your current IDE seems to be using an indentation of 1 space instead of the recommended 2

**Review by RuthNjeri — CHANGES_REQUESTED** [review]
Feedback for the change here https://github.com/publiclab/plots2/pull/9349/files#r599564420

File: app/controllers/admin_controller.rb (hunk 1 above)
Comment (BondaSaim) [reply app/controllers/admin_controller.rb]: That's done

**Review by RuthNjeri — APPROVED** [review]
Thanks @BondaSaim 🎉

**Comment by jywarren** [comment]
This looks great -- @RuthNjeri you should be able to merge this now, want to do your first merge? 🎉

**Review by cesswairimu — APPROVED** [review]
:tada:

**Comment by RuthNjeri** [comment]
Thanks @jywarren....launching in 3...2...1.. 🚀
